include/roboteam_utils/Angle.h

```diff
@@ -12,218 +12,117 @@ namespace rtt {
     class Vector2;
 
 /**
- * @brief Angle class, mostly used for angle calculation
- * 
+ * The Angle class is a modular class on the interval [-PI, PI) that deals with addition, subtractions, distances and comparisons on this scale.
+ * @author Created by: Thijs Luttikhuis <br>
+ *         Recreated by: Haico Dorenbos
+ * @since 2019-02-28
  */
-    class Angle {
+class Angle {
     public:
         /**
-         * @brief Default constructs an Angle
-         * sets angle to 0 and epsilon to 0.00001
+         * Construct the zero Angle, i.e. the Angle with 0 as angle value.
          */
         Angle() = default;
 
         /**
-         * @brief Copy constructor
-         */
-        Angle(const Angle &copy) = default;
-
-        /**
-         * @brief Construct a new Angle object
-         *
-         * @param angle Angle to set the initial angle to
+         * Construct a new Angle instance by using a double value representing the angle value, which will be directly rescaled. However you should be cautious with float values
+         * close to -MAX_FLOAT and close to MAX_FLOAT.
+         * @param angle The given double value. It is allowed to use a value outside the range [-PI, PI).
          */
         Angle(double angle);
```

Comment:
Is there a reason this is not `Angle(const double &angle)`?

Reply:
No